cruise-control/src/integrationTest/java/com/linkedin/kafka/cruisecontrol/BrokerFailureIntegrationTest.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2020 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See
```

Comment:
Nit: As a convention, CC repo uses the year the file has been created in copyright: `2020` -> `2021`.